app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/driveBar.tsx

```diff
@@ -107,6 +108,11 @@ export default function DriveBar(props: DriveBarProps) {
                         <Button
                             active
                             image={AddConnectorIcon}
+                            error={
+                                category === categorySwitcher.Category.trash
+                                    ? 'Cannot create a new data connector in Trash.'
+                                    : 'Not implemented yet.'
+                            }
```

Comment:
@PabloBuchu the "not implemented yet" is no longer correct for this branch i don't think, so this can just be `error="Cannot create a new data connector in Trash."`